config/v1/types_cluster_version.go

```diff
@@ -224,6 +227,17 @@ type UpdateHistory struct {
 // ClusterID is string RFC4122 uuid.
 type ClusterID string
 
+// ClusterVersionArchitecture enumerates valid cluster architectures.
+// +kubebuilder:validation:Enum=multi
+type ClusterVersionArchitecture string
+
+const (
+	// ClusterVersionArchitectureMulti identifies a multi architecture. A
+	// multi architecture cluster, also known as a heterogeneous cluster,
+	// is capable of running nodes with multiple architectures.
+	ClusterVersionArchitectureMulti ClusterVersionArchitecture = "multi"
```

Comment:
enumerations are CamelCase, so Multi

Reply:
Can we leave it as "multi" since that's what's being used in the payload metadata which CVO checks to set architecture? A big part of the reason to use multi rather than hetero was to avoid a disconnect here.

Reply:
> Can we leave it as "multi" since that's what's being used in the payload metadata which CVO checks to set architecture?  A big part of the reason to use multi rather than hetero was to avoid a disconnect here.

No.  Our API is consistent with kube standards to work with kube tools and match kube expectations.  There's more detail here, but we follow kube guidance regardless of the API this closely related to: https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#prefer-consistency-with-kubernetes-style-apis